src/ChunkData.h

```diff
@@ -37,7 +35,7 @@ class cChunkData
 
 	struct sChunkSection;
 
-	cChunkData(cAllocationPool<cChunkData::sChunkSection> & a_Pool);
+	cChunkData(cAllocationPool<cChunkData::sChunkSection> & a_Pool, cMemoryCounter & a_WorldMemoryCounter);
```

Comment:
This could use a bit more commenting - what are the parameters, what they're used for etc.